include/clangmetatool/tool_application_support.h

```diff
@@ -19,11 +19,16 @@ struct ToolApplicationSupport {
    * clang is installed, and that this clang-based application has been
    * installed along with the required resource files. If the installation is
    * broken, report a fatal error and exit.
+   *
+   * \param compilations Compilation Database to use for this run
+   * \param sourcePathList List of files to process during this run
+   * \param invokedArgv0 Path to the invoked clangmetatool based executable
+   *                     This must be as it appears in 'argv[0]'
    */
   static void
   verifyInstallation(const clang::tooling::CompilationDatabase &compilations,
-                     const std::vector<std::string> &sourcePathList);
-
+                     const std::vector<std::string> &sourcePathList,
+                     const std::string &invokedArgv0);
```

Comment:
Can you give this a default value of an empty string, that way the interface will be backwards compatible.